includes/payment-methods/class-wc-stripe-express-checkout-helper.php

```diff
@@ -1334,6 +1334,24 @@ public function fix_address_fields_mapping( $data ) {
 			$data['shipping_address']['address_2'] = '';
 		}
 
+		// Puerto Rico: Apple Pay and Google Pay send PR as a US state (country=US, state=PR),
```

Comment:
From my testing, this is an Apple Pay issue only.
```suggestion
		// Puerto Rico: Apple Pay send PR as a US state (country=US, state=PR),
```